ansible/vault-decrypt.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Create backup directory if it doesn't exist.
+mkdir -p vault_backup
+
```

Comment:
Since the scripts only work if you're in covidcast-indicators/ansible.
```suggestion
# Check if the current directory is not ansible
if [[ $(basename "$PWD") != "ansible" ]]; then
    # Check if ansible directory exists in the current directory
    if [[ -d "ansible" ]]; then
        cd ansible
    else
        echo "cd to covidcast-indicators/ansible to execute this script"
        exit 1
    fi
fi
```

Reply:
I think it maybe makes sense to remove the instructions from the file entirely since the scripts should be used going forward, and then add some small documentation somewhere on using the scripts.

I'm not totally against adding logic to the scripts to handle someone trying to run the script from the wrong location, but maybe it'll be simpler to add (as mentioned ☝️) some small documentation somewhere and explicitly say "you need to be in this place to use these scripts". Some small documentation for this process is probably warranted anyway.

Reply:
@minhkhul

- I removed unnecessary instruction from vault_plain.yaml
- Added a small README to the ansible directory with basic usage of the scripts

Reply:
if youre in the wrong directory, `vault.yaml` and/or `vault_plain.yaml` wont be there and the operation will fail anyway...

a couple suggestions, neither of which is truly necessary: (1) you could move the directory creation inside the `if` so it only happens when theres actually a file to back up, and (2) you could add an `echo` statement in there too to remind the user that the backup is happening.